.buildkite/pipeline.build.yml

```
@@ -591,3 +592,12 @@
       python/ray/tests/lightgbm/...
       python/ray/tests/ml_py36_compat/...
       python/ray/tests/xgboost/...
+
+
+- label: ":toolbox: CI Tools"
+  conditions: ["RAY_CI_TOOLS_AFFECTED"]
```

```suggestion
  conditions: ["RAY_CI_TOOLS_AFFECTED"]
  instance_size: small
```

Also, can we move this to `pipeline.test.yml` or do we need toolchains here?